portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
```

Comment:
Creating an `.env` file is optional, you could create the file or simply add the Environment variables in the terminal, before `yarn start`, look [here](https://github.com/3scale/porta/blob/master/portafly/README.md#environment-variables)

Reply:
yeah, there are options like adding it to your .zshrc / .bashrc, explicitely declare them before executing the command, etc

Reply:
what would be our suggested way?